2-Operating-Systems/1-arch.md

```diff
@@ -0,0 +1,271 @@
+# How to install Arch Linux: The TL;DR
+
+## Overview
+
+This guide is meant to be an opinionated and to the point guide on installing
+Arch Linux. Power users with different opinions should refer to the official
+[Arch Linux installation guide][1].
+
+[1]:https://wiki.archlinux.org/index.php/Installation_guide
+
+## Requirements
+ * A Linux computer to run early steps
+   * Can use an [Ubuntu live CD][1] if a Linux computer is not available
+ * A blank flash drive, 1GB+
+ * Target computer
+    * Known to be compatible with Linux
+    * Supports UEFI boot
+    * Has wireless card
+    * Does not require proprietary "out of tree" drivers
+
+[1]: https://www.ubuntu.com/download/desktop
+
+## Goals
+ * Full disk encryption
+ * Passwordless: Use GPG Smartcard for login, sudo, ssh
+ * Simple partitioning scheme
+
+## Steps
+
+1. Download latest Arch Linux ISO image file
+  1. Install torrent client such as aria2
```

Comment:
Because then a user has to hunt down a mirror, and manually verfify integrity and these are steps most new people get really tripped on. Plus it easily avoids downgrade attacks and the Arch maintainers say it is preferred for bandwidth (but they never actually say how to do it)

Reply:
I think more people would have trouble installing aria2 than picking a mirror from the mirror page....